server/src/main/java/org/apache/iotdb/db/engine/storagegroup/TsFileResource.java

```diff
@@ -58,6 +58,14 @@
    */
   private Map<String, Long> endTimeMap;
 
+  public TsFileProcessor getProcessor() {
```

Comment:
could we add a partitionId in the TsFileResource? then, you can compare Resource by this field after restarting

Reply:
This may be refactored in the following pr of sync. This pr may have a huge impact on sync model, sync model must be refactor after this pr.